src/Transport.ts

```diff
@@ -607,6 +613,14 @@ export default class Transport {
             if (meta.attempts < maxRetries) {
               meta.attempts++
               debug(`Retrying request, there are still ${maxRetries - meta.attempts} attempts`, params)
+
+              // exponential backoff on retries, with jitter
+              const backoff = options.retryBackoff ?? this[kRetryBackoff]
+              const backoffWait = backoff(0, 4, meta.attempts)
```

Comment:
It seems like your intention is not to expose this as a function to developers, so maybe this isn't important right now, but would it make sense to future-proof this and make the min and max arguments also configurable?

Reply:
Good question. Nobody in the community asked for retry backoff, much less the ability to provide their own algorithm, so I think we can wait until someone actually asks for it?

Reply:
Hi, this feature is add as default ... and value is not map on client of eleasticsearch, I am thinking about this feature, is it really need when we have several  elasticsearch node? why we need to wait if an elasticsearch node not provide response? the next elasticsearch will provide response no?